assets/src/edit-story/components/helpCenter/menu/tips.js

```diff
@@ -66,31 +66,37 @@ const StyledArrow = styled(Icons.ArrowAlt)`
   transform: rotate(180deg);
 `;
 
-function Tip({ children, onClick }) {
+function Tip({ children, onClick, unread }) {
   return (
     <StyledButton size={BUTTON_SIZES.SMALL} onClick={onClick}>
-      <ButtonText unread={true}>{children}</ButtonText>
+      <ButtonText unread={unread}>{children}</ButtonText>
       <StyledArrow />
     </StyledButton>
   );
 }
 Tip.propTypes = {
   children: PropTypes.node,
   onClick: PropTypes.func,
+  unread: PropTypes.bool.isRequired,
 };
 
 const TIPS_ITERABLE = Object.entries(TIPS);
-export function Tips({ onTipSelect = () => {} }) {
+export function Tips({ onTipSelect = () => {}, read }) {
   return (
     <Panel>
       {TIPS_ITERABLE.map(([key, tip]) => (
-        <Tip key={key} onClick={() => onTipSelect(key)}>
+        <Tip
+          unread={read.indexOf(key) === -1}
```

Comment:
If we make `readTips` an object, this just becomes `!readTips[key]` which is kinda nice